server/boot/certificate.js

```diff
@@ -66,7 +66,7 @@ function getIdsForCert$(id, Challenge) {
 // getFormatedDate(challengeMap: Object, challengeId: String) => String, throws
 function getFormatedDate(challengeMap, challengeId) {
   return moment(challengeMap[challengeId].completedDate)
-    .format('MMM Do, YYYY');
+    .format('MMM D, YYYY');
```

Comment:
Update this to the correct format as requested in the issue.
Refer : http://momentjs.com/docs/#/displaying/format/